tools/windows_buildenv.bat

```diff
@@ -5,7 +5,8 @@ CALL :REALPATH "%~dp0\.."
 SET MIXXX_ROOT=%RETVAL%
 
 IF NOT DEFINED PLATFORM (
-    SET PLATFORM=x64
+    ECHO ^ERROR: The PLATFORM environment variable is not defined. Please ensure you are using the correct shell like 'x64 Native Tools Command Prompt for VS 20XX
+    EXIT /B 1
```

Comment:
The breaks the call by double click. This must not depend on setting environment variables before. Either leave the old default, or add a host architecture detection.